index.html

```diff
@@ -61,22 +78,22 @@
 				// },
 				//Phường 4
 				//Example
-				// {
-				// 	type: "point",
-				// 	longitude: 106.80378931111271, 
-				// 	latitude: 10.870369163501929,
-				// 	Name: "Ghế ngồi",
-				// 	Location: "trong khuông viên trường ĐH CNTT",
-				// 	symbol: {
-				// 		type: "simple-marker",
-				// 		color: [153, 51, 51],
-				// 		outline: {
-				// 			color: [255, 255, 255],
-				// 			width: 1
-				// 		}
-				// 	},
-				// 	popupTemplate: point_template_chair
-				// },
+				{
+					type: "point",
+					longitude: 106.68127715688188, 
+					latitude: 10.774075083216818,
+					Name: "Bệnh viện Bình Dân",
+					Location: "Địa chỉ: 371 Điện Biên Phủ, Phường 4, Quận 3, Hồ Chí Minh 700000, Việt Nam",
+					symbol: {
+						type: "simple-marker",
+						color: [255, 204, 0],
+						outline: {
+							color: [255, 255, 255],
+							width: 1
+						}
+					},
+					popupTemplate: point_template_P4_hospital

```

Comment:
rename point_template_P4_hospital to something like ward_4_poppup

Reply:
i renamed to point_template_ward_4_poppup